Services/ConfigService.cs

```diff
@@ -25,6 +25,14 @@ public Uri GetLnBitsHost()
         return new Uri(host);
     }
 
+    public Uri GetWsHost()
+    {
+        var lnbitsHost = this.GetLnBitsHost();
+        var deviceId = this.GetSwitchId();
+        var wsHost = lnbitsHost.ToString().Replace("https", "wss").Replace("http", "ws");
```

Comment:
You can safely replace only the `http` by `ws` since the extra `s` is kept.

```c#
Console.WriteLine("http://casa21.space".Replace("http", "ws"));
Console.WriteLine("https://casa21.space".Replace("http", "ws"));
// ws://casa21.space
// wss://casa21.space
```

Reply:
@nGoline  clever 🧙‍♂️